swift/ql/lib/codeql/swift/generated/Comment.qll

```diff
@@ -7,6 +7,9 @@ module Generated {
   class Comment extends Synth::TComment, Locatable {
     override string getAPrimaryQlClass() { result = "Comment" }
 
+    /**
+     * Gets the text.
```

Comment:
If we can do it, I'd prefer `Gets the name of this comment.`, `Gets the name of this file.` etc?

Reply:
I think it is doable, but I have to think about implications. For example
``Gets the `index`th argument of this apply expression`` from `ApplyExpr` sounds good, but
``Gets the name of this abstract function declaration`` from `AbstractFunctionDecl` should probably drop the `abstract` part. Or also `declaration`, but then `argument of this apply` or `member of this brace` from `BraceStmt` would sound weird.

Probably just requires the `docname` modifier to be extended to classes as well, so we can fix the cases that end up not to our liking.